src/js/modules/rpc/errors.ts

```diff
@@ -134,6 +137,26 @@ const createDisableDoesNotExist: DisableCoprocResponse = (id) => ({
   disableResponseCode: DisableResponseCode.scriptDoesNotExist,
 });
 
+const validateWasmAttributes = (
+  handle: Coprocessor,
+  id: bigint,
+  logger: Logger
+): boolean => {
+  if (handle.inputTopics === undefined || handle.inputTopics.length === 0) {
```

Comment:
what if `handle` is indefined.

Reply:
yes this is important, if the entire script is bad due to syntax error i believe this can happen

Reply:
I guarantee the undefined case [here](https://github.com/vectorizedio/redpanda/pull/1620/commits/c059e4c624ca3ac708c9e8e17ef30d7b2fe02f11#diff-4eb32ee08cb0f76a194fd87da673ee41778443ddf0b62eb4c58ef9d80998bec4R221), but also I can add the validation here too